tests/src/python/test_qgsqueryresultmodel.py

```diff
@@ -56,6 +58,7 @@ def tearDownClass(cls):
     @classmethod
     def _deleteBigData(cls):
 
+        return
```

Comment:
Is this deliberate?

Reply:
Of course not, luckily the test is failing. This was a cherry pick from a development branch btw.